templates/caching-service.json

```diff
@@ -314,32 +363,62 @@
       {
          "description": "Namespace for the application. Note: The namespace is required for creating proper RoleBindings. Specifying wrong namespace will prevent cluster from forming.",
          "name": "NAMESPACE",
+         "displayName": "Namespace",
          "required": true,
          "value": "myproject"
       },
       {
          "description": "The name for the application.",
          "name": "APPLICATION_NAME",
+         "displayName": "Application Name",
          "required": true,
          "value": "caching-service-app"
       },
       {
          "description": "Infinispan image.",
          "name": "IMAGE",
          "required": true,
-         "value": "docker-registry.engineering.redhat.com/jboss-dataservices/datagrid-online-services"
```

Comment:
This doesn't look good. We should point to the same registry the CE CI is pointing to. Alternatively we could point to Red Hat external registry (but that's after GA).

Reply:
Sorry I removed this by accident. I had replaced the default value for debugging and obviously I must have removed the line instead of reverting it